src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -360,6 +390,18 @@ class ActivityFeed extends React.Component<Props, State> {
     } = this.state;
 
     const feedList = getSortedFeedData(tabs, activeTab, feedData);
+    const feedSections = [];
+
+    feedList.forEach((listItem) => {
+      const formattedDate = formatDate(new Date(listItem.createdAt * 1000), 'MMM D');
+      const existingSection = feedSections.find(({ title }) => title === formattedDate) || {};
+      if (!Object.keys(existingSection).length) {
+        feedSections.push({ title: formattedDate, data: [{ ...listItem }] });
+      } else {
+        existingSection.data.push(listItem);
```

Comment:
We are making a copy on one of the branches and using the value on the other one. I'd say we stay consistent with either option:
```js
feedSections.push({ title: formattedDate, data: [{ ...listItem }] });
existingSection.data.push({ ...listItem });
// or
feedSections.push({ title: formattedDate, data: [ listItem ] });
existingSection.data.push(listItem);
```